dlt/destinations/snowflake/__init__.py

```diff
@@ -0,0 +1,43 @@
+from typing import Type
+from dlt.common.data_writers.escape import escape_bigquery_identifier
+
+from dlt.common.schema.schema import Schema
+from dlt.common.configuration import with_config, known_sections
+from dlt.common.configuration.accessors import config
+from dlt.common.destination import DestinationCapabilitiesContext
+from dlt.common.destination.reference import JobClientBase, DestinationClientConfiguration
+from dlt.common.data_writers.escape import escape_postgres_identifier, escape_postgres_literal, escape_bigquery_identifier
+
+from dlt.destinations.snowflake.configuration import SnowflakeClientConfiguration
+
+
+@with_config(spec=SnowflakeClientConfiguration, sections=(known_sections.DESTINATION, "snowflake",))
+def _configure(config: SnowflakeClientConfiguration = config.value) -> SnowflakeClientConfiguration:
+    return config
+
+
+def capabilities() -> DestinationCapabilitiesContext:
+    caps = DestinationCapabilitiesContext()
+    caps.preferred_loader_file_format = "jsonl"
+    caps.supported_loader_file_formats = ["jsonl"]
+    caps.escape_identifier = escape_postgres_identifier
+    caps.max_identifier_length = 63
+    caps.max_column_identifier_length = 63
+    caps.max_query_length = 32 * 1024 * 1024
+    caps.is_max_query_length_in_bytes = True
+    caps.max_text_data_type_length = 1024 * 1024 * 1024
+    caps.is_max_text_data_type_length_in_bytes = True
+    caps.supports_ddl_transactions = True
+    caps.alter_add_multi_column = False
```

Comment:
It says so in the docs, but I got errors with it. I think it might be different syntax with just one `ADD COLUMN` clause, I have to try this:
```ADD COLUMN col1 NUMBER, col2 NUMBER```

Reply:
This syntax works. I extracted a separate method for the `ADD COLUMN` generation and overrode it in the snowflake client: https://github.com/dlt-hub/dlt/pull/414/commits/69abda3bd0ad21ab94a71125207711ebffe57d08